src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -1,13 +1,37 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Truck
- * Do not remove no-field constructor
- */
 public class Truck extends Machine {
+    private String liftingCapacity;
```

Comment:
Consider using more appropriate data types for the fields 'liftingCapacity' and 'power'. If these are meant to represent numerical values, using String may not be the best choice.